src/class-parsely.php

```diff
@@ -653,4 +670,93 @@ private function are_credentials_managed(): bool {
 
 		return true;
 	}
+
+	/**
+	 * Sets the values of managed options.
+	 *
+	 * @since 3.9.0
+	 * @access private
+	 */
+	private function set_managed_options(): void {
+		$managed_options = apply_filters( 'wp_parsely_managed_options', false );
+
+		if ( ! is_array( $managed_options ) ) {
+			return;
+		}
+
+		// Don't allow certain options to be set as managed.
+		unset(
+			$managed_options['apikey'],
+			$managed_options['api_secret'],
+			$managed_options['metadata_secret'],
```

Comment:
Just to confirm: we're separating "managed options" from "managed credentials"? i.e. for overriding apikey / api_secret / metadata_secret, we'd use the `wp_parsely_credentials` filter.